takwimu/dataset_release_utils.py

```diff
@@ -0,0 +1,32 @@
+from django.conf import settings
```

Comment:
Shouldn't this file be in hurumap repo altogether?

Reply:
Preferably in `data.utils`

Reply:
@kilemensi this is a workaround for only Takwimu